src/main/scala/firrtl/passes/InferWidths.scala

```diff
@@ -81,7 +93,7 @@ object InferWidths extends Pass {
 
     def remove_cycle(n: String)(w: Width): Width = {
       //;println-all-debug(["Removing cycle for " n " inside " w])
-      w map remove_cycle(n) match {
```

Comment:
Is this related to the Issue.  It's hard for me to see how it is.  Maybe a comment on how this works